app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html

```diff
@@ -144,27 +166,31 @@ <h2 class="heading-medium">@messages("landingPage.reportingDisposals.h2")</h2>
-        <li>@messages("landingPage.capasitorOrPersonalRepresentative.list2.bullet1")</li>
-        <li>@messages("landingPage.capasitorOrPersonalRepresentative.list2.bullet2")</li>
-        <li>@messages("landingPage.capasitorOrPersonalRepresentative.list2.bullet3")</li>
-    </ul>
-    <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p5"))</p>
-    <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p6"))</p>
-    <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p7"))</p>
-    <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p8"))</p>
-    <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.example.title"))</p>
-    <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p9"))</p>
-    <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p10"))</p>
-    <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p11"))</p>
+    @if(capacitorsAndPersonalRepresentativesJourneyEnabled) {
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.enabled.p2", appConfig.returnsForSomeoneWhoHasDied))</p>
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p3"))</p>
+        <ul class="list-bullet">
+            <li>@messages("landingPage.capasitorOrPersonalRepresentative.list1.bullet1")</li>
+            <li>@messages("landingPage.capasitorOrPersonalRepresentative.list1.bullet2")</li>
+            <li>@messages("landingPage.capasitorOrPersonalRepresentative.list1.bullet3")</li>
+            <li>@messages("landingPage.capasitorOrPersonalRepresentative.list1.bullet4")</li>
+            <li>@Html(messages("landingPage.capasitorOrPersonalRepresentative.list1.bullet5", appConfig.powerOfAttorney))</li>
+        </ul>
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p4"))</p>
+        <ul class="list-bullet">
+            <li>@messages("landingPage.capasitorOrPersonalRepresentative.list2.bullet1")</li>
+            <li>@messages("landingPage.capasitorOrPersonalRepresentative.list2.bullet2")</li>
+            <li>@messages("landingPage.capasitorOrPersonalRepresentative.list2.bullet3")</li>
+        </ul>
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p5"))</p>
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p6"))</p>
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p7"))</p>
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p8"))</p>
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.example.title"))</p>
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p9"))</p>
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p10"))</p>
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p11"))</p>
+    } else {
+        <p>@Html(messages("landingPage.capasitorOrPersonalRepresentative.p2", appConfig.contactHmrc))</p>
```

Comment:
did you disable capacitors/personal reps and compare with the existing live version to check that it's still the same?

Reply:
Yes. Done it for both.